### DISCUSSION TITLE
Hide metadata in posts

### DISCUSSION THREAD

**Discussion by e-alizadeh** [quarto-dev/quarto-cli#2446] [Q&A]
I want to control the metadata shown on the listings as well as on blog pages. 
Is there a way (a flag) to NOT show AUTHOR in a post?

![quarto_sample_blog](https://user-images.githubusercontent.com/30320825/190305564-fb8d0095-49d0-4b5e-a94e-91d26bdefa01.png)

**Comment by dragonstyle** [comment]
Right now, anytime the author is present, it is shown. You can omit the author and the title block will not output the author. If you have a more complex scenario, we'd love to hear about it.

We do have an open issue requesting the ability to suppress an author (so an author can be provided in a shared `_metadata.yml` file or project and then override to empty for a specific post). Right now we don't really support this well though we should soon.

**Reply by e-alizadeh** [reply]
Thanks. It makes sense.

**Comment by Almeus** [comment]
Not sure if this was ever addressed, but I have a similar scenario. I have a group of .qmd files. I want to add a description to the metadata block of each .qmd file that will show up on a listing page but not on the page itself.

In general, I'd like a way to suppress metadata that is normally rendered on the page but that may be useful on other parts of a site.

**Reply by mcanouil** [reply]
You can use CSS to hide the content you don't want `display: none;`.
Use your browser inspect/developer mode to see what you should target.
